src/views/new evaluation/dropDownSelector.js

```diff
@@ -0,0 +1,43 @@
+const DropdownSelector = function (options = {}) {
+    
+        const arr = [`<legend class="field-title field-title-skin">${setKey(options.legends)}</legend>`];
+    
+        const constant = options.items.map((el) => {
+    
+            const dropDownOptions = el.options.map(e => `<option value="${e}">${e}</option>`);
+    
+            const dropdownEl = dropDownOptions.join('');
+    
+            const forNameId = setId(options.legends, el.label);
+    
+            return `
+    
+                <ul>
+    
+                 <li>
+    
+                    <label for="${forNameId}">${el.label}</label>
+    
+                    <select name="${forNameId}" id="${forNameId}">
+    
+                    ${dropdownEl}
+    
+                    </select> 
+    
+                  </li>
+    
+                </ul>
+    
+       `;
+    
+        },
+    
+        );
+    
+    
+    
+        const selectOption = arr.push(constant.join(''));
```

Comment:
While this _will_ work and is good for experimenting with array APIs, it's probably overkill. Simple string concatenation would have been better suited in this particular case.